csrankings-j.csv

```diff
@@ -2055,7 +2055,7 @@ Joseph Jean Boutros,Texas A&M at Qatar,http://www.josephboutros.org,lyjl3nMAAAAJ
 Joseph K. Kearney,University of Iowa,http://homepage.cs.uiowa.edu/~kearney,wKC0dwcAAAAJ
 Joseph K. Liu,Monash University,http://users.monash.edu.au/~kailiu,GGe9ON8AAAAJ
 Joseph Kee-Yin Ng,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~jng,NOSCHOLARPAGE
-Joseph Keshet,Bar-Ilan University,http://u.cs.biu.ac.il/~jkeshet,GoWgJ1AAAAAJ
+Joseph Keshet,Technion,https://keshet.net.technion.ac.il,GoWgJ1AAAAAJ

```

Comment:
ECE, not CS. Please clarify (see instructions).

Reply:
Dear Emery,
Faculty members from ECE (previously EE) are allowed to supervise graduate students in computer science (see https://github.com/emeryberger/CSrankings/pull/1517).
Please let me know if further information is needed.
Best,
Joseph